tests/stub/homedb/test_homedb.py

```diff
@@ -8,6 +8,9 @@
 
 
 class TestHomeDb(TestkitTestCase):
+
+    required_features = types.Feature.BOLT_4_4,
```

Comment:
Would it make sense to exclude this from the test methods?

Reply:
How do you mean?

Reply:
Since all test methods seem to require 4.4, they probably do not need individual declarations.
See changes in `tests/stub/session_run_parameters/test_session_run_parameters.py` for example.

Reply:
:+1: